python/cleanup/main.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 13 08:56:23 2017
+
+@author: Tatiana Kim
+"""
+
+import pandas as pd
+import numpy as np
+import utilities as ut
+import transform_records as tr
+import time as tm
+
+
+# import data 
+rawData = pd.read_csv('C:/Users/Tatiana Kim/Dropbox (Iridium Insights)/Local Folder Tatiana/ML/ParkingProject/BANES_Historic_Car_Park_Occupancy.csv') 
```

Comment:
Can we make this a relative path please. E.g.
`../../data/BANES_Historic_Car_Park_Occupancy.csv`

It easier to create this folder than setting up windows with a user called Tatiana and a dropbox folder for Iridium Insights :)

Reply:
Yes, that makes sense :) Done in the latest commit 8db4cd1a905c7ef7b965233e77a3f4edd108c8bb